src/sentry/monitors/models.py

```diff
@@ -23,9 +24,12 @@
 )
 from sentry.db.models.utils import slugify_instance
 from sentry.locks import locks
-from sentry.models import Environment, Project
+from sentry.models import Environment
 from sentry.utils.retries import TimedRetryPolicy
 
+if TYPE_CHECKING:
+    from sentry.models.group import Project
```

Comment:
this is to prevent a circular import dependency

Reply:
Why are you importing from `.group` instead of `.project`?

Reply:
oh whoops